cibuild.sh

```diff
@@ -72,7 +72,7 @@ setMonoDir(){
 SKIP_BOOTSTRAP=false
 
 # Paths
-THIS_SCRIPT_PATH="`dirname \"$0\"`"
+THIS_SCRIPT_PATH=`pwd`
```

Comment:
Yeah I was mostly trying to test my theory that the build was broken because the path wasn't absolute.  "pwd" gives an absolute path and showed that the build would work.  I'll find a way to get the absolute path to the script's directory...